debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -1497,4 +1492,23 @@ public PreparedStatement readTablePreparedStatement(CommonConnectorConfig connec
     public String quotedTableIdString(TableId tableId) {
         return tableId.toDoubleQuotedString();
     }
+
+    /**
+     * Prepares qualified column names with appropriate quote character as per the specific database's rules
+     *
+     * @param tableId
+     * @param columnName
+     * @return formatted string
+     */
+    public String quotedColumnIdString(TableId tableId, String columnName) {
+        String tableName = tableId.table();
+        if (tableName.contains("\"")) {
```

Comment:
@gunnarmorling, regarding the quote in a table name, it won't work.

Suppose we have a table `order"table` and its column `id` will be referenced as `order""table`.`id` after quoting, but in actual the table would be still `order"table`.

So for SELECT `order""table`.`id` FROM `order"table`; it would throw SQLException as `Unknown column 'order""table.id' in 'field list'`.

The same thing will happen for column names too. I think the double-quote checking and modification for both table and column names should only be a part of the Postgres connector. Wdyt?

Reply:
Yes, let's remove this logic from here then and override `quotedColumnIdString()` in the Postgres specific connection type then:

    @Override
    public String quotedColumnIdString(TableId tableId, String columnName) {
        String tableName = tableId.table();
        if (tableName.contains("\"")) {
            tableName = tableName.replaceAll("\"", "\"\"");
        }
        if (columnName.contains("\"")) {
            columnName = columnName.replaceAll("\"", "\"\"");
        }

        return super(tableId, columnName);
    }

Can you push a commit for this? Theoretically, we may want to do the same for the other connectors (using their quoting/escaping scheme), but I think this is all rather hypothetical at this point and we can address it as a follow-up if truly needed. Let's keep it for Postgres though, given we have the tests for it already.

Reply:
@gunnarmorling addressed the above suggestions!

Reply:
Cool, LGTM now.